src/main/java/net/minecraftforge/fmp/ForgeMultipart.java

```diff
@@ -18,11 +18,11 @@
 import net.minecraftforge.fmp.network.MultipartNetworkHandler;
 import net.minecraftforge.fmp.util.MCMPEventHandler;
 
-@Mod(modid = ForgeMultipart.MODID, name = ForgeMultipart.NAME, version = ForgeMultipart.VERSION, acceptedMinecraftVersions = "[1.9,1.9.4]")
+@Mod(modid = ForgeMultipart.MODID, name = ForgeMultipart.NAME, version = ForgeMultipart.VERSION)
 public class ForgeMultipart
 {
     
```

Comment:
The entire @Mod shouldnt be needed anymore if it's going internal to forge then it's not a mod.